SurfaceDuoSamples/build.gradle

```diff
@@ -4,10 +4,8 @@
  *
  */
 
-plugins {
-    id 'com.android.application'
-    id 'kotlin-android'
-}
+apply plugin: 'com.android.application'
+apply plugin: 'kotlin-android'
```

Comment:
Why was the gradle plugins syntax changed?

Reply:
Align with the way we used it in the SDK